pkg/system/phase4_configuring.go

```diff
@@ -437,6 +445,35 @@ func (r *Reconciler) ReconcileDefaultBackingStore() error {
 	return nil
 }
 
+func (r *Reconciler) preparePVoolBackingStore() error {
+
+	// create secret
+	o := util.KubeObject(bundle.File_deploy_internal_secret_empty_yaml)
+	secret := o.(*corev1.Secret)
+	secret.Name = fmt.Sprintf("backing-store-%s-%s", nbv1.StoreTypePVPool, "noobaa-default-backing-store")
+	secret.Namespace = options.Namespace
+	secret.StringData = map[string]string{}
+	secret.Data = nil
+
+	if !util.KubeCreateSkipExisting(secret) {
+		log.Fatalf(`❌ Could not create Secret %q in Namespace %q (conflict)`, secret.Name, secret.Namespace)
+	}
+
+	// create backing store
+	r.DefaultBackingStore.Spec.Type = nbv1.StoreTypePVPool
+	r.DefaultBackingStore.Spec.PVPool = &nbv1.PVPoolSpec{}
+	r.DefaultBackingStore.Spec.PVPool.Secret.Name = secret.Name
+	r.DefaultBackingStore.Spec.PVPool.Secret.Namespace = secret.Namespace
+	r.DefaultBackingStore.Spec.PVPool.StorageClass = ""
```

Comment:
Do we have to provide a storageclass? it doesn't work without it? I think it suppose to take the default in that case...

Reply:
checked it and it works fine without providing anything :)